cmake/modules/AddSwift.cmake

```diff
@@ -359,9 +359,10 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl"
+        "-ldl" "-llog" "-latomic"
         "${SWIFT_ANDROID_NDK_PATH}/sources/cxx-stl/llvm-libc++/libs/armeabi-v7a/libc++_shared.so")
     list(APPEND library_search_directories
+        "${SWIFT_ANDROID_NDK_PATH}/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/arm-linux-androideabi/lib/armv7-a"
```

Comment:
I mean the `4.9` in 

```bash
"${SWIFT_ANDROID_NDK_PATH}/toolchains/arm-linux-androideabi-4.9"
```

right?
 
If it can't and I'm just blowing smoke here tell me and I'll get this CI'd and merged as-is.  But as it stands I really don't have a way to properly test this patch and I'm trying not to take anything that looks too user-specific.